src/platform/silabs/rs911x/wfx_sl_ble_init.cpp

```diff
@@ -40,8 +37,10 @@ SilabsBleWrapper::BleEvent_t bleEvent;
  */
 void SilabsBleWrapper::rsi_ble_on_mtu_event(rsi_ble_event_mtu_t * rsi_ble_mtu)
 {
-    bleEvent.eventType = BleEventType::RSI_BLE_MTU_EVENT;
-    memcpy(&bleEvent.eventData->rsi_ble_mtu, rsi_ble_mtu, sizeof(rsi_ble_event_mtu_t));
+    SilabsBleWrapper::BleEvent_t bleEvent;
+    bleEvent.eventType                  = BleEventType::RSI_BLE_MTU_EVENT;
+    bleEvent.eventData.connectionHandle = 1;
+    memcpy(&bleEvent.eventData.rsi_ble_mtu, rsi_ble_mtu, sizeof(rsi_ble_event_mtu_t));

```

Comment:
Something like this make the code much cleaner in my opinion. This can be applied to each function that create a bleEvent.
```suggestion
SilabsBleWrapper::BleEvent_t bleEvent = {
    .eventType = BleEventType::RSI_BLE_MTU_EVENT,
    .eventData = {
        .connectionHandle = 1,
        .rsi_ble_mtu = *rsi_ble_mtu
    }
};
```